### PR TITLE
docs: usage: MNIST: Update use case to normalize image arrays

### PR DESCRIPTION
**Pull request by sakshamarora1**
Fixes: #672 

Signed-off-by: Saksham Arora sakshamarora1001@gmail.com

### CODE REVIEW

File: docs/usage/mnist.rst [hunk 1]
```diff
@@ -23,26 +23,23 @@ with ``pip``.
 
     $ pip install -U dffml-model-tensorflow
 
-Train the model.
+Create a dataflow config file which will be used by the :py:class:`DataFlowSource <dffml.source.df.DataFlowSource>` 
+to normalize the data just before feeding it to the model.
 
-.. literalinclude:: /../examples/MNIST/train.sh
+.. literalinclude:: /../examples/MNIST/normalize.yaml
```
[the remark this answers is not in the document]
Comment (pdxjohnny) [reply docs/usage/mnist.rst:29]: Crap I just realized that I'd forgotten to hit submit review, sorry about the lag time!

[the remark this answers is not in the document]
Comment (sakshamarora1) [reply docs/usage/mnist.rst:29]: Oh okay! I'll add the `dataflow create` command.

File: examples/MNIST/create_dataflow.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+dffml dataflow create multiply associate_definition -config yaml \
+    -seed product=associate_spec 0.00392156862=multiplier_def | \
+    sed -e 's/value: product/value:\n    image: product/g' \
+    -ze 's/multiplicand:\n      - seed/multiplicand:\n      - seed:\n        - image/g' | \
+    tee normalize.yaml
```
Comment (pdxjohnny) [inline examples/MNIST/create_dataflow.sh:5]: For https://github.com/intel/dffml/issues/661

```
dffml dataflow create multiply associate_definition -config yaml \
    -seed '{"image": "product"}'=associate_spec 0.00392156862=multiplier_def \
    -flow '["seed":["image"]]'=multiply.inputs.multiplicand | \
    tee normalize.yaml 
```

File: docs/usage/mnist.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (pdxjohnny) [reply docs/usage/mnist.rst:36]: Let's also add it before running `train.sh` in `examples/MNIST/test_mnist.py`

File: examples/MNIST/create_dataflow.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+dffml dataflow create multiply associate_definition -config yaml \
+    -seed product=associate_spec 0.00392156862=multiplier_def | \
```
Comment (pdxjohnny) [inline examples/MNIST/create_dataflow.sh:2]:
```suggestion
    -seed '{"image": "product"}'=associate_spec 0.00392156862=multiplier_def | \
```

Comment (pdxjohnny) [inline examples/MNIST/create_dataflow.sh:2]:
```suggestion
    -seed '{"image": "product"}'=associate_spec 0.00392156862=multiplier_def | \
```

**Pull request merged**